Sources/Core/EthereumABI/ABIParsing.swift

```diff
@@ -172,12 +170,6 @@ extension ABI.Input {
         let indexed = self.indexed == true
         return ABI.Element.Event.Input(name: name, type: parameterType, indexed: indexed)
     }
-
-    func parseForError() throws -> ABI.Element.EthError.Input {
-        let name = self.name ?? ""
-        let parameterType = try ABITypeParser.parseTypeString(self.type)
-        return ABI.Element.EthError.Input(name: name, type: parameterType)
-    }
```

Comment:
`ABI.Element.EthError.Input` was replaced with `ABI.Element.InOut` since these are two identical structures.
No need for `func parseForError() throws -> ABI.Element.EthError.Input` then.